modules/post/osx/gather/enum_osx.rb

```diff
@@ -459,7 +386,7 @@ def get_keychains(log_folder)
         end
       end
     else
-      current_user = cmd_exec("/usr/bin/id -nu").chomp
+      current_user = cmd_exec("/usr/bin/id -nu")
       print_status("Enumerating and Downloading keychains for #{current_user}")
       keychain_files = cmd_exec("usr/bin/security list-keychains").split("\n")
```

Comment:
Not related to this PR, but `usr/bin/security list-keychains` should almost certainly be prefaced with `/`